pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/output.gif}}`
 
-- Make a GIF animation with gifsicle:
+- Unoptimize a GIF in place:
 
-`gifsicle --delay={{10}} --loop *.gif > {{anim.gif}}`
+`gifsicle -b {{path/to/input.gif}} --unoptimize`
 
-- Extract frames from an animation:
+- Extract a frame from an animation:
 
-`gifsicle {{anim.gif}} '#0' > {{firstframe.gif}}`
+`gifsicle {{path/to/input.gif}} '#{{0}}' > {{path/to/firstframe.gif}}`
 
-- You can also edit animations by replacing, deleting, or inserting frames:
+- Make a GIF animation from selected still/animated GIFs:
 
-`gifsicle -b {{anim.gif}} --replace '#0' {{new.gif}}`
+`gifsicle {{*.gif}} --delay={{10}} --loop > {{path/to/input.gif}}`
+
+- Reduce file size using multiple methods at the expense of quality:
+
+`gifsicle -b {{path/to/input.gif}} --optimize=3 --lossy={{100}} --colors={{16}} --dither`
+
+- Delete the first 10 frames and all frames after #20 from an animation. There are editing options to insert, append and replace as well:
```

Comment:
```suggestion
- Delete the first 10 frames and all frames after #20 from an animation.:
```

Although useful, it is not the place.

Reply:
There seems to be some [upstream issue](https://github.com/kohler/gifsicle/issues/186) where these flags can't be used together. So what I was trying to imply with that line was that the `--delete` flag could pretty easily be replaced with any of the other three editing options.

Reply:
I'll just submit an edit in the future with a line that uses multiple flags at once, when the issue is fixed.

Reply:
you could change the command to {{--delete|--append|--insert|--replace}}

Reply:
No problem. The flags aren't *quite* interchangeable, even though they're similar. That was a good idea though.